java-client/src/main/java/org/opensearch/client/opensearch/_types/mapping/KnnVectorMethod.java

```diff
@@ -0,0 +1,229 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
```

Comment:
If it is not copied from Elasticsearch, please remove this header (for brand new files we need only OpenSearch license header)